travis-ymls/retrying.py_travis.yml

```diff
@@ -0,0 +1,42 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : retrying.py
+# Source Repo         : https://github.com/rholder/retrying.git
+# Travis Job Link     : https://travis-ci.com/github/sreekanth370/retrying/builds/212652856
+# Created travis.yml  : No
+# Maintainer          : sreekanth reddy <bsreekanthapps@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+language: python
+arch:
+ - amd64
+ - ppc64le
+python:
+  - 2.6
+  - 2.7
+  - 3.2
+  - 3.3
+  - 3.4
+  - 3.5
+  - pypy
+ # Disable unsupport version 2.6,3.2, 3.3 & pypy on ppc64le
+jobs: 
+  exclude:
+    - arch: amd64
```

Comment:
Can we do not do something like below and avoid excluding 2.6, 3.2 and 3.3 for both ?
python:
    - 2.7
    - 3.4
    - 3.5
    - pypy